### ISSUE TITLE
doc: awful.screen marked as deprecated?!

### ISSUE THREAD

**Issue by blueyed** [awesomeWM/awesome#1146]
`awful.screen` is documented as being deprecated (http://new.awesomewm.org/apidoc/libraries/awful.screen.html), but then in http://new.awesomewm.org/apidoc/classes/screen.html there are a lot of references to `awful.screen` functions.
I think this is confusing.

As for this (and other) deprecated aliased docs: they should not be used in the main navigation: you tend to click on `awful.screen` only to see that it's deprecated.

**Comment by Elv13** [comment]
The rational for this was that have `client` and `awful.client` confused users. I have seen that, many, many time on IRC. So I merged the pages together (client, screen, tag, mouse, etc). So now there is 1 page per concept and an united list of functions. I also merged `awful.client.focus` and other submodules into their parent pages (but in different doc sections to keep the separation).

The reason why I created these aliases was that the user might look for "awful.screen" in the index and be confused if its not there. If you think it isn't worth it, then I am fine with deleting them, along with other useless pages like `objects.screen`. Then there is the question of other deprecated modules like `awful.wibox` that has been renamed to `awful.wibar` and the widgets that were moved around. I kept aliases for them too, but they are not really necessary if you think `awful.screen` is not.
